src/styles/StyleUtils.js

```diff
@@ -420,7 +420,14 @@ function getModalPaddingStyles({
     modalContainerStyleMarginBottom,
     modalContainerStylePaddingTop,
     modalContainerStylePaddingBottom,
+    insets,
 }) {
+    // set fallback value for safeAreaPaddingBottom to keep padding bottom consistent with padding top.
+    // More info: issue #17376
+    if (insets.bottom === 0) {
+        // eslint-disable-next-line no-param-reassign
+        safeAreaPaddingBottom = modalContainerStylePaddingTop;
```

Comment:
Lets use a different variable here, so we can avoid disabling the lint rule here